packages/slice-machine/lib/builders/SliceBuilder/SetupDrawer/index.tsx

```diff
@@ -111,30 +116,46 @@ const HelpSection = () => (
         flexDirection: "column",
       }}
     >
-      <Flex
-        sx={{
-          alignItems: "center",
-          mb: 3,
-        }}
-      >
-        <Flex
-          sx={{
-            mr: 2,
-          }}
-        >
+      <Flex sx={{ alignItems: "center", mb: 3 }}>
+        <Flex sx={{ mr: 2 }}>
           <FaRegQuestionCircle size={20} color="textGray" />
         </Flex>
-        <Text sx={{ fontSize: 2, fontWeight: 500 }}>Help Section</Text>
+        <Text sx={{ fontSize: 2, fontWeight: 500 }}>{heading}</Text>
       </Flex>
-      <Text sx={{ color: (t) => t.colors?.textClear }}>
-        Are you having difficulties setting up the preview? You can check{" "}
-        <Link target={"_blank"} href={"https://prismic.io"}>
-          the documentation
-        </Link>{" "}
-        for more info to set it up correctly.
-      </Text>
+      <Text sx={{ color: (t) => t.colors?.textClear }}>{children}</Text>
     </Flex>
   </Flex>
 );
 
+const HelpSection = () => (
```

Comment:
funny thing but I think that this disappear in kevin's PR about preview setup.

Reply:
On top of that, I think this part about the setup drawer has been rewritten so better merge that PR quickly 😂